tests/sources/fixtures/github/connector.json

```diff
@@ -29,28 +29,14 @@
                 },
                 "host": {
                         "depends_on": [{"field": "data_source", "value": "github_server"}],
-                        "label": "Server URL",
+                        "label": "GitHub URL",
                         "order": 2,
                         "type": "str",
                         "value": "http://127.0.0.1:9091"
                 },
-                "auth_method": {
-                        "display": "dropdown",
-                        "label": "Authentication method",
-                        "options": [
-                            {"label": "Personal access token", "value": "personal_access_token"},
-                            {"label": "GitHub App", "value": "github_app"}
-                        ],
-                        "order": 3,
-                        "type": "str",
-                        "value": "personal_access_token"
-                },
```

Comment:
Why's this being removed? I still see it in the `get_default_configuration` for github.py

Reply:
Initally, I have [the change to connector.json](https://github.com/elastic/connectors/pull/2374/files) in this PR, then I extracted the change to a separate PR, and reverted [here](https://github.com/elastic/connectors/pull/2372/commits/7a6b8877c1f88e0a8f1e56fe19e7e4b966374d9f) in this PR.

I guess this is because of the merge of PR https://github.com/elastic/connectors/pull/2374? I will try to update branch and see if it can remove the change.

Reply:
:sweat_smile: No it's still there, I will merge this PR and address it in a separate PR.

Reply:
Follow-up PR https://github.com/elastic/connectors/pull/2389